core/src/test/scala/unit/kafka/utils/ReplicationUtilsTest.scala

```diff
@@ -34,7 +34,7 @@ class ReplicationUtilsTest extends ZooKeeperTestHarness {
   private val controllerEpoch = 1
   private val isr = List(1, 2)
   private val topicPath = s"/brokers/topics/$topic/partitions/$partition/state"
-  private val topicData = Json.encode(Map("controller_epoch" -> controllerEpoch, "leader" -> leader,
+  private val topicData = Json.legacyEncodeAsString(Map("controller_epoch" -> controllerEpoch, "leader" -> leader,
```

Comment:
We should use `encodeAsString` here since this test has been converted to `KafkaZkClient`.

Reply:
Done.